boards/nxp/frdm_mcxn236/doc/index.rst

```diff
@@ -74,6 +74,11 @@ The FRDM-MCXN236 board configuration supports the following hardware features:
 +-----------+------------+-------------------------------------+
 | LPCMP     | on-chip    | sensor(comparator)                  |
 +-----------+------------+-------------------------------------+
+| FLEXIO    | on-chip    | flexio                              |
++-----------+------------+-------------------------------------+
+| DISPLAY   | on-chip    | flexio; MIPI-DBI. Tested with       |
```

Comment:
Can you add this board to the display testcase, like we did here for MCXN947: https://github.com/zephyrproject-rtos/zephyr/blob/69b73ebb0ab9b85fd2f219fa6ca1e135ba58ac88/samples/drivers/display/sample.yaml#L88

Reply:
Thanks for reminder, updated.